docs/content/tutorial/saving-your-data.mdx

```diff
@@ -76,11 +75,110 @@ defs = Definitions(
   information.
 </Note>
 
-To see the effect of this change, reload your code location in the Dagster UI and materialize your assets. When you click on an asset, you’ll see the `path` metadata pointing to the `data` directory on your computer.
+Now, you will modify your three assets `topstory_ids`, `topstories`, and `most_frequent_words` to use the I/O manager to read and write data rather than direct calls to the `json` and `pandas` libraries. To do this, you will:
+
+- Return your data from each asset, rather than manually writing it to the file system. The I/O manager will now save the data.
+- Modify how you define dependencies between assets. To use the I/O manager to load the data associated with an upstream asset into memory, you use the upstream asset's name as the name of one of the arguments to the downstream asset's function.
```

Comment:
A few thoughts on this bit:

- Should the dependencies bullet come first, since that's what comes first in the code examples?
- Perhaps bold the first sentences of each bullet to make them stand out, ex: "**Modify how you define dependencies between assets.** To use..."
- I think this does a good job of concisely explaining the changes, tbh. What do you feel is missing or falling short? We can chat about it later if Zoomin' is easier too

Reply:
re ordering: I've been going back and forth on this, and eventually went with the current order bc chronologically data needs to be saved before it can be loaded. but also the dependency setup happens at the top of the asset and return happens at the bottom so it also makes sense the other way around. idk idk 

re what i thought was falling short: I didn't have any specifics, i just had to rewrite this section several times and didn't have an "ah ha i got it" moment. also had tired friday brain so wasn't confident what i wrote made any sense. After reading it again today i think it isn't so bad

Reply:
The ordering makes sense; thanks for explaining it. I think this is fine as-is, and agreed that the wording in here is also fine. Let's plan to revisit if we get feedback that something needs clarification.